### PR TITLE
[DSD-6443]: correct the issuers list in the issuers-config

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: mimoto-issuers-config.json [hunk 1]
```diff
@@ -228,127 +228,33 @@
...
-      ],
-      "additional_headers": {
-        "Accept": "application/json"
-      },
-      ".well-known": "https://${mosip.injicertify.mosipid.host}/v1/certify/issuance/.well-known/openid-credential-issuer?version=vd11",
+      "client_id": "${mimoto.mock.oidc.clientid}",
+      "client_alias": "mpartner-mock-testing",
+      "scopes_supported": ["mosip_identity_vc_ldp"],
+      "additional_headers": { "Accept": "application/json" },
+      ".well-known": "https://${mosip.esignet.mock.host}/.well-known/openid-credential-issuer?version=v11",
```
Comment (swatigoel) [inline mimoto-issuers-config.json]: this should be `mock.identity.esignet.host`

File: mimoto-issuers-config.json [hunk 2]
```diff
@@ -132,7 +132,7 @@
         }
       ],
       "protocol": "OpenId4VCI",
-      "client_id": "${mimoto.oidc.partner.clientid}",
+      "client_id": "${mimoto.mosipid.oidc.clientid}",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:135]: mimoto.oidc.mosipid.partner.clientid

File: mimoto-issuers-config.json [hunk 3]
```diff
@@ -212,7 +212,7 @@
         }
       ],
       "protocol": "OpenId4VCI",
-      "client_id": "${mimoto.oidc.sunbird.partner.clientid}",
+      "client_id": "${mimoto.insurance.oidc.clientid}",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:215]: mimoto.oidc.insurance.partner.clientid

File: mimoto-issuers-config.json [hunk 4]
```diff
@@ -228,127 +228,33 @@
...
-      "client_id": "${mimoto.oidc.partner.clientid}",
-      "client_alias": "mpartner-default-mimotooidc",
-      "scopes_supported": [
-        "mosip_identity_vc_ldp"
-      ],
-      "additional_headers": {
-        "Accept": "application/json"
-      },
-      ".well-known": "https://${mosip.injicertify.mosipid.host}/v1/certify/issuance/.well-known/openid-credential-issuer?version=vd11",
+      "client_id": "${mimoto.mock.oidc.clientid}",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:245]: mimoto.oidc.mock.partner.clientid

File: mimoto-issuers-config.json [hunk 5]
```diff
@@ -228,127 +228,33 @@
...
-      "authorization_audience": "https://${mosipid.identity.esignet.host}/v1/esignet/oauth/v2/token",
-      "token_endpoint": "https://${mosip.api.public.host}/residentmobileapp/get-token/MosipCertify",
-      "proxy_token_endpoint": "https://${mosipid.identity.esignet.host}/v1/esignet/oauth/v2/token",
-      "credential_endpoint": "https://${mosip.injicertify.mosipid.host}/v1/certify/issuance/vd11/credential",
-      "credential_type": [
-        "VerifiableCredential",
-        "MOSIPVerifiableCredential"
-      ],
-      "credential_audience": "https://${mosip.injicertify.mosipid.host}",
+      "authorization_audience": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
```
Comment (swatigoel) [inline mimoto-issuers-config.json]: mock.identity.esignet.host

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: mimoto-issuers-config.json (hunk 1 above)
Comment (vijay151096) [reply mimoto-issuers-config.json]: updated

File: mimoto-issuers-config.json (hunk 3 above)
Comment (vijay151096) [reply mimoto-issuers-config.json]: updated

File: mimoto-issuers-config.json (hunk 5 above)
Comment (vijay151096) [reply mimoto-issuers-config.json]: updated

File: mimoto-issuers-config.json (hunk 4 above)
Comment (vijay151096) [reply mimoto-issuers-config.json]: updated

File: mimoto-issuers-config.json (hunk 2 above)
Comment (vijay151096) [reply mimoto-issuers-config.json]: updated

**Pull request merged**